### PR TITLE
feat(arc-509): link cp admins for archief idp

### PR DESCRIPTION
**Pull request by bartnaessens**
- link cp admins for archief idp
- renamed getAuth method for consistency with HetArchief controller

Issues: ARC-758 & ARC-509

### CODE REVIEW

File: src/modules/auth/controllers/het-archief.controller.ts [hunk 1]
```diff
@@ -163,6 +164,14 @@ export class HetArchiefController {
 				}
 			}
 
+			// CP_ADMIN: Link the user to the maintainer
+			if (userGroup === Group.CP_ADMIN) {
```
Comment (bertyhell) [inline src/modules/auth/controllers/het-archief.controller.ts:168]: also for kiosk users
or do you want to do that in a separate PR?

**Review by bertyhell — APPROVED** [review]

File: src/modules/auth/controllers/het-archief.controller.ts (hunk 1 above)
Comment (bartnaessens) [reply src/modules/auth/controllers/het-archief.controller.ts]: Updated in this PR

**Review by bertyhell — APPROVED** [review]
